.github/workflows/check-patch.yml

```diff
@@ -59,11 +67,11 @@ jobs:
 
 
   build-el9:
-
+    # Build only on master
```

Comment:
This comment is redundant as the code below already states as much. Please add a comment explaining the reasons behind this decision.

Reply:
fixed comment